build.zig

```diff
@@ -7,10 +7,16 @@ pub fn build(b: *std.Build) !void {
     const vulkan_dep = b.dependency("vulkan_zig_generated", .{});
```

Comment:
Looks like we're missing it here too

Reply:
vulkan_zig_generated doesn't need those options

Reply:
I disagree, every call to `b.dependency` should specify those options just to be on the safe side